example_project/example_project/orchestra_settings.py

```diff
@@ -68,11 +68,11 @@ def setup_orchestra(settings_module_name):
     # Tasks and Workflows
     ######################
 
-    # Installed orchestra workflows
-    settings.ORCHESTRA_PATHS = (
-        ('simple_workflow.workflow', 'simple_workflow'),
-        ('journalism_workflow.workflow', 'journalism_workflow'),
+    # Installed orchestra workflows to be included as Django apps
```

Comment:
Two quick ones hidden in http://orchestra.readthedocs.org/en/stable/getting_started.html#create-a-django-project
- Make sure you added simple_workflow to your INSTALLED_APPS setting following the previous section.
- Add 'simple_workflow' to settings.INSTALLED_APPS in the “General” section if you want to run the demo workflow (instructions below), and add 'journalism_workflow' if you want to run the journalism workflow.